wandb/apis/public.py

```diff
@@ -4235,9 +4235,30 @@ def id(self):
     def file_count(self):
         return self._attrs["fileCount"]
 
+    @property
+    def source_version(self):
+        """
+        Returns:
+            (str) The artifact's version index under its parent artifact collection. This will return
+            a string with the format "v{number}".
+        """
+        return f"v{self._sequence_version_index}"
+
     @property
     def version(self):
-        return "v%d" % self._version_index
+        """
+        Returns:
+            (str): The artifact's version index under the given artifact collection. This will return
+            a string with the format "v{number}".
+        """
+        for a in self._attrs["aliases"]:
```

Comment:
are you always guaranteed to have the `aliases` key?

Reply:
Yes, the `aliases` is always going to exist for an artifact version because it'll always contain at least the version index of the artifact.